backend/db/schema.sql

```diff
@@ -15,31 +15,12 @@ CREATE TABLE IF NOT EXISTS files(
     id INTEGER PRIMARY KEY NOT NULL,
     filename TEXT NOT NULL,
     filetype TEXT NOT NULL CHECK (filetype IN ('movie', 'song', 'episode', 'picture')),
-    filesize INTEGER NOT NULL,
-    created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
-    expired TIMESTAMP,
-    description TEXT,
+    fileSize INTEGER NOT NULL,
```

Comment:
See the differences here between your file version and my updated one and fix

Reply:
Added new commit with the fix version